backend-api/src/functions/asyncBiometricToken/validateRequestBody/validateRequestBody.ts

```diff
@@ -0,0 +1,89 @@
+import { errorResult, Result, successResult } from "../../utils/result";
+
+export function validateRequestBody(
+  body: string | null,
+): Result<IAsyncBiometricTokenValidParsedRequestBody> {
+  if (body == null) {
+    return errorResult({
+      errorMessage: `Request body is either null or undefined. Request body: ${body}`,
```

Comment:
Nitpick: do we need to give the value of the body/field in all of these error messages? In most cases, the message already says what it is (null, undefined, missing, empty string). It's really just the document type where it'd be particularly interesting to see the value they provided, I think.

Reply:
Removed where all values are clear via message already